### ISSUE TITLE
SC4E data is missing many packages

### ISSUE THREAD

**Issue by Zasco** [sebamarynissen/sc4pac-helpers#4]
In [**sc4e.js**](https://github.com/sebamarynissen/sc4pac-helpers/blob/main/lib/data/sc4e.js).
Just PRed two in #3.
I realized `36 : lowkee33:seasonal-flora-patch-maxis` also is missing. Led me to believe there is more.
I use these for my [browser extension project](https://github.com/Zasco/sc4pac-browser-plugin).

**Comment by sebamarynissen** [comment]
Yeah, the list is incomplete because I only fill in new ones "on the go" whenever I run into them when adding packages to the default channel (such as in https://github.com/memo33/sc4pac/pull/50 or https://github.com/memo33/sc4pac/pull/54)
Feel free to create PRs for them filling in the gaps!

**Comment by memo33** [comment]
Note that you can obtain these IDs directly from the [sc4pac-channel-contents.json](https://memo33.github.io/sc4pac/channel/sc4pac-channel-contents.json) file:
```json
{
 "packages": [
  {
   "group": "lowkee33",
   "name": "seasonal-flora-patch-maxis",
   "externalIds": { "sc4e": ["36"] },
   …
  },
  …
```

**Comment by Zasco** [comment]
> Note that you can obtain these IDs directly from the [sc4pac-channel-contents.json](https://memo33.github.io/sc4pac/channel/sc4pac-channel-contents.json) file:
> 
> ```json
> {
>  "packages": [
>   {
>    "group": "lowkee33",
>    "name": "seasonal-flora-patch-maxis",
>    "externalIds": { "sc4e": ["36"] },
>    …
>   },
>   …
> ```

Great! Is this new? I don't recall having seen this.

**Comment by memo33** [comment]
It's quite new. It was added with sc4pac 0.5.0.

**Comment by Zasco** [comment]
> It's quite new. It was added with sc4pac 0.5.0.

Great! I must have overlooked it. I'll change my logic as this is more complete/reliable.
